guillotina/db/storages/cockroach.py

```diff
@@ -184,6 +197,16 @@ async def has_unique_constraint(self, conn):
             logger.warning("Unknown error attempting to detect constraints installed.", exc_info=True)
         return False
 
+    async def delete(self, txn, oid):
+        conn = await txn.get_connection()
+        sql = self._sql.get("CR_TRASH_PARENT_ID", self.objects_table_name)
+        deleted_id = uuid.uuid4().hex
```

Comment:
This is super weird. Why is this necessary?

The constraints should be composite for parent_id, id so you shouldn't need to change the id right?

Reply:
Any doubt ? can you review ?